### DISCUSSION TITLE
JetBrains Fleet is available

### DISCUSSION THREAD

**Discussion by worksofliam** [codefori/vscode-ibmi#905] [General]
![image](https://user-images.githubusercontent.com/3708366/195351801-f46bc39a-6cc3-4704-b653-fa70afc425f8.png)

Tried it out this morning. It's ok, but sadly doesn't do a lot for me right now. No out of the box Node.js debugging, but the UI certainly is very nice. I like the minimalistic look and the chosen fonts.

Also, there is no plugin/extension support yet. Sad times.

**Comment by Wright4i** [comment]
JetBrains Fleet was just a solution to a problem JetBrains created themselves, over segmentation of their products. They didn't have an all-in-one code editor to compete with vscode. I do admit it's a pretty IDE. Reminds me of Sublime vs Atom. Atom had the looks but Sublime was much more extensible at the time.
